SevenSlots/SevenSlots/SevenSlots/ViewModel/BlackjackViewModel.cs

```diff
@@ -574,11 +563,10 @@ public void GameOver()
         }
         public void SaveData()
         {
-            _player.BankRoll += _player.TotalBet;
-            _player.TotalBet = 0;
-            //_playerData.Add(_player);
-            //_gameBusiness.SavePlayer(_playerData, _player.Name);
-            //_playerData.Clear();
+            User.Wallet = _player.BankRoll;
+            _userService.patchWallet(User.Id, User.Wallet);
+
+            //TestUser; testpass -> credentials for testing; don't delete yet!
```

Comment:
huh?

Reply:
I need those for when I login to the app, since I don't know how or where to see the database :)

Reply:
Ok, I'll approve for now I guess, but please don't forget to delete that when you're done.

Reply:
Still need it?